setup.sh

```diff
@@ -1,6 +1,7 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-easy_install --user requests
-easy_install --user requests[security]
+pip install requests
```

Comment:
Since easy_install used the `--user` flag, we should use `pip install --user` here just in case pip wants to install the packages in a system directory that the user doesn't have access to.

Reply:
I agree. It would be preferable if not using a virtualenv. Unfortunately, the `--user` option fails when using a virtualenv. I'll look into changing some things to get the best of both worlds.